### DISCUSSION TITLE
Storefront menu settings empty after update / outage (horizon)

### DISCUSSION THREAD

**Discussion by brahima** [swellstores/community#203] [Help] [answered]
Hello,

Seems that i'm hitting [the same issue as here](https://github.com/orgs/swellstores/discussions/200) (see code below) with the horizon theme. I first posted in this discussion but since it was answered, i think it won't get any visibility.
In fact i was actually developing early in the morning (6AM / Paris Time Zone) when the API outage occurred ... so i had to do other things.

Despite several hours have passed, i still have the same issue.
I tried creating another test store ... same error.
My stores are (kotoboka-dev and kotoboka-test-1)

TypeError: Cannot read properties of undefined (reading 'menu') at formatStoreSettings (webpack-internal:///./utils/settings.ts:72:39) at getStoreSettings (webpack-internal:///./lib/shop/fetchingFunctions.ts:323:79) at processTicksAndRejections (node:internal/process/task_queues:96:5) at async eval (webpack-internal:///./lib/utils/fetch_decorators.ts:23:56)

Any help would be greatly appreciated.

**Comment by logeshswell** [comment, marked as answer]
It is fixed and please check the discussion comment [here](https://github.com/orgs/swellstores/discussions/200#discussioncomment-5547262). Can you confirm if it's working fine?

**Reply by brahima** [reply]
Hello @logeshswell and thanks.
It's ok now
